sdk/src/Services/S3/Custom/Transfer/Internal/DownloadDirectoryCommand.cs

```diff
@@ -206,7 +196,7 @@ private void ValidateRequest()
                 throw new InvalidOperationException("The LocalDirectory Specified is null or empty!");
             }
 
-            if (File.Exists(this._request.S3Directory))
+            if (File.Exists(this._request.LocalDirectory))
```

Comment:
i think this was a bug. we were checking if s3 directory existed on machine and not local directory???

Reply:
do we need a dedicated dev config because of this line? @philasmar @normj ?

Reply:
Since we are fixing an unrelated bug to the feature we should have a change log entry for it.

I'm curious if this should have been `Directory.Exists` instead of `File.Exists`. Also was the validation basically broken before and users have been successfully working when the directory already exists. This is fixing a bug but might be introducing a breaking change that might not be okay.

Reply:
in that case ill remove this line change from this PR and we can update later if we want